docs/en/latest/admin-api.md

```diff
@@ -541,7 +541,7 @@ In addition to the basic complex equalization algorithm selection, APISIX's Upst
 |type            |required|the balancer algorithm||
 |nodes           |required, can't be used with `service_name` |Hash table or array. If it is a hash table, the key of the internal element is the upstream machine address list, the format is `Address + (optional) Port`, where the address part can be IP or domain name, such as `192.168.1.100:80`, `foo.com:80`, etc. The value is the weight of node. If it is an array, each item is a hash table with key `host`/`weight` and optional `port`/`priority`. The `nodes` can be empty, which means it is a placeholder and will be filled later. Clients use such an upstream will get 502 response. |`192.168.1.100:80`|
 |service_name    |required, can't be used with `nodes` |the name of service used in the service discovery, see [discovery](discovery.md) for more details|`a-bootiful-client`|
-|discovery_type  |required, if `service_name` is used | the type of service discovery, see [discovery](discovery.md) for more details|`eureka`|
+|discovery_type  |required, if `service_name` is used | the type of service discovery, see [discovery](discovery.md) for more details|`consul_kv`, `eureka`, `dns`, `nacos`|
```

Comment:
This cell is just an example. You need to add the types after "the type of service discovery, see [discovery](discovery.md) for more details".

Reply:
> This cell is just an example. You need to add the types after "the type of service discovery, see [discovery](discovery.md) for more details".

ok, my bad.